src/widgets/include/SmartPeak/ui/Widget.h

```diff
@@ -403,6 +399,7 @@ namespace SmartPeak
   */
   static void showQuickHelpToolTip(const std::string& ui_element_name)
   {
+    return;
```

Comment:
Is this correct?

Reply:
oh, good catch, I put it this because I was hunting a bug by reducing ImGui widgets.